seagence/assets/monitors/defect_detection_monitor.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "Seagence Defect Detection Monitor",
+	"type": "event-v2 alert",
+	"query": "events(\"source:my_apps status:error service:seagence\").rollup(\"count\").last(\"5m\") > 0",
```

Comment:
the source should be the source name of the integration, not my apps. Please update to reflect what the customer will use